test/plugin_functional/test_suites/embedding_visualizations/embed_by_id.js

```diff
@@ -36,6 +37,7 @@ export default function ({ getService }) {
     await retry.try(async () => {
       await testSubjects.waitForDeleted('visLoadingIndicator');
     });
+    await delay(1000);
```

Comment:
Is this meant to be in there? If so why? Also we should use `PageObjects.common.sleep` instead, to not create more dependencies onto the underlying libraries (that we are planning to replace).

Reply:
the test was flaky. needs to be further investigated